blocklytest/src/androidTest/java/com/google/blockly/model/BlockTestStrings.java

```diff
@@ -59,7 +59,7 @@
             + "    {"
             + "      \"type\": \"field_checkbox\","
             + "      \"name\": \"NAME\","
-            + "      \"checked\": true"
+            + "      \"checked\": TRUE"
```

Comment:
FWIW, I prefer allowing/keeping the JSON boolean value instead of the string form.